Spigot-Server-Patches/0446-UUID-support-in-CraftProfileBanList.patch

```diff
@@ -0,0 +1,85 @@
+From 95b343c1556c2acfc47a9102f8bfe6cb676a0a88 Mon Sep 17 00:00:00 2001
+From: pop4959 <pop4959@gmail.com>
+Date: Wed, 16 Oct 2019 18:59:15 -0700
+Subject: [PATCH] UUID support in CraftProfileBanList
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/CraftProfileBanList.java b/src/main/java/org/bukkit/craftbukkit/CraftProfileBanList.java
+index 84a1f9c8..6494029c 100644
+--- a/src/main/java/org/bukkit/craftbukkit/CraftProfileBanList.java
++++ b/src/main/java/org/bukkit/craftbukkit/CraftProfileBanList.java
+@@ -14,6 +14,7 @@ import org.apache.commons.lang.Validate;
+ 
+ import com.google.common.collect.ImmutableSet;
+ import com.mojang.authlib.GameProfile;
++import java.util.UUID; // Paper
```

Comment:
Use fully qualified class names instead of imports, eases maitenance.

Reply:
Got it, will do!